modules/aruco/src/aruco.cpp

```diff
@@ -1782,9 +1787,18 @@ void drawDetectedMarkers(InputOutputArray _image, InputArrayOfArrays _corners,
 /**
  */
 void drawAxis(InputOutputArray _image, InputArray _cameraMatrix, InputArray _distCoeffs, InputArray _rvec,
-              InputArray _tvec, float length)
+              InputArray _tvec, float length, int thickness)
 {
-    drawFrameAxes(_image, _cameraMatrix, _distCoeffs, _rvec, _tvec, length, 3);
+    vector<Point3f> axis;
+    axis.push_back(Point3f(0.f, 0.f, 0.f));
+    axis.push_back(Point3f(length, 0.f, 0.f));
+    axis.push_back(Point3f(0.f, length, 0.f));
+    axis.push_back(Point3f(0.f, 0.f, -length));
```

Comment:
why `-length`? Your function differs from standard drawFrameAxes with opposite direction of the third axis.

Reply:
This was added to improve appearance. The drawAxis() function doesn't make sense, as it is just calls drawFrameAxes().